src/status_im/contact/db.cljs

```diff
@@ -65,7 +65,8 @@
 (defn public-key->new-contact [public-key]
   {:name        (gfycat/generate-gfy public-key)
    :address     (ethereum/public-key->address public-key)
-   :photo-path  (identicon/identicon public-key)
+   :photo-path  ""
```

Comment:
why not removing it rather than using empty string?

Reply:
It's up to us, in this case I prefer having an empty string as it makes it easier to interact with status-go and database (strings are not nullable), plus is consistent type (always string, never null), but can change if you feel strongly about it.

Reply:
in the clojure code having an empty string here is not idiomatic, the photo-path is not there and "" is truthy, for consistency I don't think we should do this. If there is a need for an empty string at the borders of the program it can be added there